.aspell.en.pws

```diff
@@ -341,3 +341,28 @@ optimizations
 structs
 CompactSize
 encodings
+bigsize
+BigSize
+golang
+json
+testReadVarInt
+testWriteVarInt
+ReadVarInt
+WriteVarInt
+varIntTest
+tlv
+ExpErr
+Fatalf
+fc
```

Comment:
It's a bit weird to have to add all of those in the spell-checker...
I think that if we always wrap them in back-ticks (e.g. `0xfc`) the spell-checker doesn't look at them, which might be a better solution.

Reply:
good point, i gave a shot at modifying the spell checker to ignore code blocks, but not sure that's the best approach

Reply:
i'm +1 for ignoring code blocks fwiw